### PR TITLE
Not reversing orders array in place because that's bad

### PR DESCRIPTION
**Pull request by alfetopito**
Follow up to https://github.com/gnosis/dex-react/pull/853#discussion_r406006081

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr870--dexreact.review.gnosisdev.com

File: src/reducers-actions/orders.ts [hunk 1]
```diff
@@ -51,10 +51,10 @@ export const reducer = (state: OrdersState, action: ReducerActionType): OrdersSt
       const { orders: currentOrders } = state
 
       // reverse new orders
-      newOrders.reverse()
+      const reversed = newOrders.slice(0).reverse()
```
Comment (anxolin) [inline src/reducers-actions/orders.ts:54]: ubernit: `newOrdersReversed`? so it's more clear when you read line 57

File: src/reducers-actions/orders.ts [hunk 2]
```diff
@@ -40,9 +40,9 @@ export const reducer = (state: OrdersState, action: ReducerActionType): OrdersSt
 
       // default sorting order is ascending by order id
       // since we are starting from 0, reverse to have the latest on top
-      orders.reverse()
+      const reversed = orders.slice(0).reverse()
```
Comment (Velenir) [inline src/reducers-actions/orders.ts:43]: You don't need `0` in `array.slice()`, not that it matters

Comment (alfetopito) [reply src/reducers-actions/orders.ts:43]: I read on [the internet this is faster](https://stackoverflow.com/a/21514254/1272513), so it must be true.

Comment (Velenir) [reply src/reducers-actions/orders.ts]: Must be true then, no other way around it. Internet is the final authority

Comment (alfetopito) [reply src/reducers-actions/orders.ts]: Exactly! You get me.

**Pull request merged**